src/node.cc

```diff
@@ -4910,6 +4943,8 @@ int Deinitialize() {
 
   deinitialize::_DeinitV8();
 
+  // TODO(js): Do we need to tear down OpenSsl?
```

Comment:
I am by no means an expert on OpenSSL, nor on how it is being used by node (for now, at least), but from what I have seen in [their wiki](https://wiki.openssl.org/index.php/Library_Initialization#Cleanup), there are functions for OpenSSL cleanup.

Considering we want as little memory leaks as possible (we will have to live with some from [the OpenSSL library itself](https://rt.openssl.org/Ticket/Display.html?id=2561&user=guest&pass=guest)), I think this should be investigated, but probably in a new issue+PR, and not here. It seems like a bunch of research into OpenSSL and too big for this Issue.